server/seeders/seeds.js

```diff
@@ -9,41 +9,41 @@ db.once('open', async () => {
   await Room.deleteMany({});
   await Reservation.deleteMany({});
 
-  // create user data
-  const userData = [];
+  // // create user data
```

Comment:
could you _uncomment_ the seed data before pushing your changes?

Reply:
The seed data needs to be updated by hand now.  Will add to Issue list.